src/test/java/gov/healthit/chpl/aqa/asserts/ViewProductPageAsserts.java

```diff
@@ -0,0 +1,83 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.junit.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
```

Comment:
Can we settle on one or the other of these two packages? I don't care which one, but the expected/actual orders are different in their respective assertEquals, and we shouldn't be using two packages that do basically the same thing. @ashwinimore , do you have a preference?

Reply:
I would prefer assertTrue

Reply:
Sorry, I meant the use of the junit and testng packages. They both have "assertTrue" and "assertEquals", but we shouldn't be using both junit & testng in the same project